chart/index.yaml

```diff
@@ -3,69 +3,79 @@ entries:
   jenkins-operator:
```

Comment:
Please revert index.yaml and regenerate it with command `make helm-deploy` https://github.com/jenkinsci/kubernetes-operator/blob/master/Makefile#L502.